app/blob_storage.py

```diff
@@ -0,0 +1,70 @@
+import re
+from functools import lru_cache
+
+from app.api_clients.azure import AzureBlobServiceClient
+from app.conf import get_settings
+
+
+@lru_cache
+def get_azure_blob_service_client() -> AzureBlobServiceClient:
```

Comment:
A function that merely instantiates a class and does nothing else is typically unnecessary because it introduces an extra layer of function call overhead without adding any real value.

Furthermore, decorating this function with lru_cache is counterproductive because:

1) The function always receives the same argument so the cache will store only one instance of the class.
2) creating an instance of Azure BlobServiceClient is not an expensive operation, then caching it does not provide a meaningful performance improvement. Instead, it just adds the overhead of maintaining the LRU cache.